### DISCUSSION TITLE
Unraid + nslookup not working

### DISCUSSION THREAD

**Discussion by devanteweary** [netalertx/NetAlertX#638] [Q&A]
Hey guys, just tried out the Unraid container for NetAlertX. (in host mode)

Long story short, nslookup isn't working for hostname returns.

I use pihole at 192.168.1.10.  
I've put that nameserver in the resolv.conf but it just never resolves.

If I console into the container, I can ping my laptop at .100, my router at .1, and various other devices on the network.  
However, I cannot ping Pihole at .10.

Anyone else ever get that issue?

Oh and the odd thing is if I do a Device > Tools > NSLOOKUP... it shows time outs for .10 but then sees the hostname anyway (but doesn't report it in the name field).

![image](https://github.com/jokob-sk/NetAlertX/assets/29805497/e98defa1-70e4-42e8-8902-a4307429f97d)

**Comment by jokob-sk** [comment]
Hi there,

I don't run unpaid so maybe unrelated, but I did have to enable reverse DNS on AdGuard:

https://github.com/jokob-sk/NetAlertX/blob/main/docs/REVERSE_DNS.md
